### PR TITLE
Read geometries with ST_AsEWKB

### PR DESCRIPTION
**Pull request by elemoine**
This PR is an alternative to https://github.com/geoalchemy/geoalchemy2/pull/121.

### CODE REVIEW

File: geoalchemy2/types.py [hunk 1]
```diff
@@ -100,12 +104,12 @@ def get_col_spec(self):
...
 
     def column_expression(self, col):
-        return func.ST_AsBinary(col, type_=self)
+        return getattr(func, self.as_binary)(col, type_=self)
 
     def result_processor(self, dialect, coltype):
         def process(value):
             if value is not None:
-                return WKBElement(value, srid=self.srid)
+                return WKBElement(value, srid=self.srid, extended=True)
```
Comment (elemoine) [inline geoalchemy2/types.py]: `extended` should be `false` in the case of geographies.

Comment (elemoine) [inline geoalchemy2/types.py]: Fixed.

**Comment by elemoine** [comment]
@loicgasser, do you want to test this more?

**Comment by loicgasser** [comment]
Yes I can give it try and install it in my projects, but it seems good.

**Comment by loicgasser** [comment]
let in you know very soon

**Comment by loicgasser** [comment]
all good for me

**Comment by elemoine** [comment]
@tsauerwein, @fredj, tell me if you want to have a look at this, otherwise I'll merge this soon.

@lukas-gitl, this PR is going to break your hack (shown in https://github.com/geoalchemy/geoalchemy2/issues/93). So we definitely need to find a better solution for you.

**Comment by elemoine** [comment]
@loicgasser, thanks for the original patch and the good collaboration on this!

**Comment by tsauerwein** [comment]
Are updates to the docs or changelog required for this change?

**Comment by elemoine** [comment]
Thanks for your comments @tsauerwein. I fixed the test, and I updated the documentation. For the changelog I will update it when preparing the next release.

**Comment by elemoine** [comment]
Merged. Thank you all for the work on this! I'll make a release next week.

**Comment by loicgasser** [comment]
:+1:

**Comment by elemoine** [comment]
0.3.0.dev1 is released on pypi.

**Comment by elemoine** [comment]
@loicgasser, have you been using 0.3.0.dev1? Should we consider releasing 0.3.0 final?

**Comment by loicgasser** [comment]
Yes we're currently working with it in several projects and it works fine.

**Pull request merged**